client/sites/marketing/tools/marketing-features-data.ts

```diff
@@ -0,0 +1,100 @@
+import page from '@automattic/calypso-router';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { translate } from 'i18n-calypso';
+import fiverrLogo from 'calypso/assets/images/customer-home/fiverr-logo.svg';
+import rocket from 'calypso/assets/images/customer-home/illustration--rocket.svg';
+import earnIllustration from 'calypso/assets/images/customer-home/illustration--task-earn.svg';
+import wordPressLogo from 'calypso/assets/images/icons/wordpress-logo.svg';
+import { marketingConnections } from 'calypso/my-sites/marketing/paths';
+import * as T from 'calypso/types';
+import { MarketingToolsFeatureData } from './types';
+
+export const getMarketingFeaturesData = (
+	selectedSiteSlug: T.SiteSlug | null,
+	recordTracksEvent: ( event: string ) => void
+): MarketingToolsFeatureData[] => [
+	{
+		title: translate( 'Let our WordPress.com experts build your site!' ),
+		description: translate(
+			"Hire our dedicated experts to build a handcrafted, personalized website. Share some details about what you're looking for, and we'll make it happen."
+		),
+		imagePath: wordPressLogo,
+		buttonText: translate( 'Get started' ),
+		buttonHref: localizeUrl( 'https://wordpress.com/website-design-service/?ref=tools-banner' ),
```

Comment:
Same as `translate`, I would move `localizeUrl` to the component.

Reply:
I will work on it, I think is worth leaving this data as raw as possible

Reply:
I have moved it as part of [361fd58](https://github.com/Automattic/wp-calypso/pull/96542/commits/361fd588e11270e18c8543241e0c5d7e4a77e882), I have tested with a Spanish site, and the URL has been correctly localized to `https://wordpress.com/es/website-design-service/?ref=tools-banner`